.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: CI
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  MacOS:
+    runs-on: macos-latest
+    defaults:
+      run:
+        shell: bash
+    env:
+      PYTHON_VERSION: '3.7'
```

Comment:
it's worth trying a bump to 3.8 here, it would be good if we could test the latest here

Reply:
Should I work on a matrix to have both?

Reply:
No I think that's overkill. We already test "old" compat on Travis